pkg/networkpath/traceroute/runner.go

```diff
@@ -249,3 +303,28 @@ func getHostname(ipAddr string) string {
 	}
 	return currHost
 }
+
+func createGatewayLookup() (network.GatewayLookup, uint32) {
+	rootNs, err := rootNsLookup()
+	if err != nil {
+		log.Errorf("failed to create gateway lookup")
+	}
+	defer rootNs.Close()
```

Comment:
We are not expected to **return** on error here?

If rootNs is nil, call to `defer rootNs.Close()` could panic?

Reply:
Yes, should be a return here, good catch, thanks